lib/galaxy/config/__init__.py

```diff
@@ -507,15 +523,19 @@ def _process_config(self, kwargs):
         self.pbs_dataset_path = kwargs.get('pbs_dataset_path', "")
         self.pbs_stage_path = kwargs.get('pbs_stage_path', "")
 
-        _sanitize_whitelist_path = self._in_managed_config_dir(self.sanitize_whitelist_file)
-        if not os.path.isfile(_sanitize_whitelist_path):  # then check old default location
-            deprecated = self._in_root_dir('config/sanitize_whitelist.txt')
-            if os.path.isfile(deprecated):
-                log.warning("The path '%s' for the 'sanitize_whitelist_file' config option is "
-                    "deprecated and will be no longer checked in a future release. Please consult "
-                    "the latest version of the sample configuration file." % deprecated)
-                _sanitize_whitelist_path = deprecated
-        self.sanitize_whitelist_file = _sanitize_whitelist_path
+        _sanitize_allowlist_path = self._in_managed_config_dir(self.sanitize_allowlist_file)
+        if not os.path.isfile(_sanitize_allowlist_path):  # then check old default location
+            for deprecated in (
+                    self._in_managed_config_dir('sanitize_whitelist.txt'),
+                    self._in_root_dir('config/sanitize_whitelist.txt')
+                ):
+                if os.path.isfile(deprecated):
+                    log.warning("The path '%s' for the 'sanitize_allowlist_file' config option is "
+                        "deprecated and will be no longer checked in a future release. Please consult "
+                        "the latest version of the sample configuration file." % deprecated)
+                    _sanitize_allowlist_path = deprecated
```

Comment:
If both paths point to a file (i.e., the one in root may have been left over by mistake), then the second path will override the first (i.e., we have the same file in `[managed_config_dir]/` and in `[root]/config` - so the one in `root`  is used, although the one in `managed_config_dir` is, likely, the more recent version. So maybe swap the paths in lines 529-30?

Reply:
Doesn't the `break` on 537 take care of that?

Reply:
@natefoo  yeah, of course it does! I simply chose not to notice it...